site/src/Librecores/ProjectRepoBundle/Entity/Project.php

```diff
@@ -323,6 +325,49 @@ public function __construct()
         $this->classifications = new ArrayCollection();
     }
 
+    /**
+     * Normalizes the object into an array of scalars|arrays.
```

Comment:
https://manual.phpdoc.org/HTMLSmartyConverter/HandS/phpDocumentor/tutorial_tags.inlineinheritdoc.pkg.html

No need to repeat doc from libraries